### PR TITLE
Fix support for hyphens

### PR DESCRIPTION
**Pull request by theofidry**
https://github.com/nelmio/alice/pull/355 introduced a BC which is removing support for the usage of hyphens (`-`) in fixture names. This commit revert that change and add tests for it.

Closes https://github.com/nelmio/alice/issues/421

/cc @Soullivaneuh

### CODE REVIEW

File: src/Nelmio/Alice/Instances/Populator/Populator.php [hunk 1]
```diff
@@ -89,7 +89,8 @@ public function populate(Fixture $fixture)
 
             $value = $property->requiresUnique() ?
                 $this->generateUnique($fixture, $property) :
-                $this->processor->process($property, $fixture->getSetProperties(), $fixture->getValueForCurrent());
+                $this->processor->process($property, $fixture->getSetProperties(), $fixture->getValueForCurrent())
+            ;
```
Comment (Soullivaneuh) [inline src/Nelmio/Alice/Instances/Populator/Populator.php]: I'm not sure this was wanted.

**Comment by Soullivaneuh** [comment]
Works like a charm, thank! :heart: 

What about deprecation notices?

**Comment by theofidry** [comment]
@Soullivaneuh they are still deprecated, unless it proves to be very easy, I don't plan to support it in v3, at least for the first iteration.

**Comment by Soullivaneuh** [comment]
Yes, I'm talking about deprecation notices! 😉

Sullivan SENECHAL

**Comment by theofidry** [comment]
Not sure what you mean, hyphens are properly deprecated: https://github.com/nelmio/alice/blob/2.x/src/Nelmio/Alice/Fixtures/Fixture.php#L391

**Pull request merged**